apps/verticalhome/js/pin_app_navigation.js

```diff
@@ -9,255 +9,256 @@
 
-					this.prevVerticalKey = 'up';
+          this.prevVerticalKey = 'up';
 
 
-					break;
+          break;
 
-				case 'ArrowDown':
+        case 'ArrowDown':
 
-					this.elements[this.selectedElemIndex].classList.remove('selected');
-					if (this.selectedElemIndex == (this.elements.length - 3)){
-						if (!this.elemList.dataset.scrolldown){
-							this.elemList.style.top = parseFloat(this.elemList.style.top) - 6 + 'rem';
-							this.elemList.dataset.scrolldown = this.elemList.style.top;
-						}
+          this.elements[this.selectedElemIndex].classList.remove('selected');
+          if (this.selectedElemIndex == (this.elements.length - 3)) {
+            if (!this.elemList.dataset.scrolldown) {
+              this.elemList.style.top = parseFloat(this.elemList.style.top) - 6 + 'rem';
+              this.elemList.dataset.scrolldown = this.elemList.style.top;
+            }
 
-						this.selectedElemIndex --;
+            this.selectedElemIndex--;
 
-						var removed = this.elements.splice(0, 1);
+            var removed = this.elements.splice(0, 1);
 
-						var cloned = removed[0].cloneNode(true);
+            var cloned = removed[0].cloneNode(true);
 
-						removed[0].classList.add('removed');
+            removed[0].classList.add('removed');
 
-						this.elemList.appendChild(cloned);
-						this.elements[this.elements.length] = cloned;
+            this.elemList.appendChild(cloned);
+            this.elements[this.elements.length] = cloned;
 
-						this.selectedElemIndex ++;
-						this.elements[this.selectedElemIndex].classList.add('selected');
+            this.selectedElemIndex++;
+            this.elements[this.selectedElemIndex].classList.add('selected');
 
-						this.elemList.style.top = this.elemList.dataset.scrolldown;
-					} else {
-						this.selectedElemIndex ++;
-						this.elements[this.selectedElemIndex].classList.add('selected');
+            this.elemList.style.top = this.elemList.dataset.scrolldown;
+          } else {
+            this.selectedElemIndex++;
+            this.elements[this.selectedElemIndex].classList.add('selected');
 
-						this.elemList.style.top = parseFloat(this.elemList.style.top) - 6 + 'rem';
-						this.elemList.dataset.scrolldown = this.elemList.style.top;
-					}
+            this.elemList.style.top = parseFloat(this.elemList.style.top) - 6 + 'rem';
+            this.elemList.dataset.scrolldown = this.elemList.style.top;
+          }
 
-					this.prevVerticalKey = 'down';
+          this.prevVerticalKey = 'down';
 
-					if (this.selectedElemIndex == 1){
-						this.elemList.style.top = '0rem';
-					}
+          if (this.selectedElemIndex == 1) {
+            this.elemList.style.top = '0rem';
+          }
 
-					break;
+          break;
 
-				case 'Accept':
+        case 'Accept':
 
-					window.removeEventListener('keydown', this);
-					app.getAppByURL(this.elements[this.selectedElemIndex].dataset.manifesturl).launch();
+          window.removeEventListener('keydown', this);
```

Comment:
To open moreApps it is needed such code

 window.removeEventListener('keydown', this);

          if (this.elements[this.selectedElemIndex].getAttribute('id') == "moreApps") {
            app.showMoreApps();
          } else {
            app.getAppByURL(this.elements[this.selectedElemIndex].dataset.manifesturl).launch();
          }